### DISCUSSION TITLE
posts/exploring-adjacency-lists-and-matrices

### DISCUSSION THREAD

**Discussion by giscus** [ganderzz/ganderzz.github.io#105] [Announcements]
# posts/exploring-adjacency-lists-and-matrices

In  we covered how we can store a DAG (Directed Acyclic Graph) in a relational database.
We looked at one representation of a graph data structure called an adjacency list—a table with two columns representing how nodes connect.
The columns store connections from the  or  node to a  or  node.Using P...

https://www.dylanpaulus.com/posts/exploring-adjacency-lists-and-matrices

**Comment by Paddy3118** [comment]
The word "unidirected" you use above should be undirected. You added an extra I after the n.

**Reply by ganderzz** [reply]
Thanks for the correction. Updated!
